_topic_map.yml

```diff
@@ -1209,7 +1209,10 @@ Topics:
   File: routing-optimization
 - Name: What huge pages do and how they are consumed by apps
   File: what-huge-pages-do-and-how-they-are-consumed-by-apps
----
+  Name: Performance-addon operator for low latency nodes
```

Comment:
@StephenJamesSmith these lines are not correctly formatted. Could you please fix, looking at the existing entries and resubmit? Otherwise this will not make it for 4.4.

Reply:
Corrected (what I believe was) the error on line 1215.

Reply:
@StephenJamesSmith Thanks! Looks like we're still seeing build errors. Did you push the latest changes?